wcivf/settings/base.py

```diff
@@ -183,7 +184,7 @@
 EMAIL_SIGNUP_ENDPOINT = (
     "https://democracyclub.org.uk/mailing_list/api_signup/v1/"
 )
-EMAIL_SIGNUP_API_KEY = ""
+EMAIL_SIGNUP_API_KEY = "f00b42"
```

Comment:
Is this correct? I had assumed that this would be added in the deploy. If this was just for when running locally its better to add to your local.py settings file rather than commit here

Reply:
It's a fake key that appears throughout our products http://bada55.io/f00b42

Reply:
Removed.